docs/source/stats.rst

```diff
@@ -0,0 +1,42 @@
+Stats
```

Comment:
Request to rename file to `array-stats.rst`, just to keep with the `module-topic` convention.